dev/src/dev/debug_qp_viewers.cljs

```diff
@@ -0,0 +1,21 @@
+(ns dev.debug-qp-viewers
+  (:require
+   [clojure.string :as str]
+   [portal.ui.api]))
+
+(defn- format-sql [sql]
```

Comment:
Placeholder until I figure out how to get https://www.npmjs.com/package/sql-formatter working here